### PR TITLE
Fix #6158: Fivetran Connector Added

### LINKED ISSUE #6158: Fivetran Connector
Let's work on ingesting pipeline data from https://www.fivetran.com/

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Fix #6158: Fivetran Connector Added

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

**Comment by harshach** [comment]
@ulixius9 are we getting any lineage details out of fivcetran

**Comment by ulixius9** [comment]
@harshach, yes we are getting the lineage details between data sources, only one thing that we missing is the job or sync run status, I couldn't find any api to fetch job status

File: catalog-rest-service/src/main/resources/json/schema/entity/services/connections/pipeline/fivetranConnection.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    "apiSecret": {
+      "title": "Fivetran API Secret",
+      "description": "Fivetran API Secret.",
+      "type": "string",
+      "format": "password"
+    },
+    "limit": {
+      "title": "Fivetran API Limit",
+      "description": "Fivetran API Limit For Pagitation.",
```
Comment (pmbrull) [inline catalog-rest-service/src/main/resources/json/schema/entity/services/connections/pipeline/fivetranConnection.json:43]: typo in pagination

**Review by pmbrull — APPROVED** [review]

**Comment by pmbrull** [comment]
@ulixius9 one last thing that I believe it is missing in this PR is adding the docs for the connector! Let me know if you have any doubts there

**Review by pmbrull — APPROVED** [review]
Looks great, thanks for the help

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**